src/PerfView/PerfView.csproj

```diff
@@ -255,13 +251,97 @@
       <Link>amd64\HeapDump.exe.config</Link>
       <Visible>False</Visible>
     </EmbeddedResource>
+    <EmbeddedResource Include="$(NuGetPackageRoot)\System.Buffers\4.5.1\lib\net461\System.Buffers.dll">
```

Comment:
⚠️ You shouldn't assume this package is in this location. Add an explicit PackageReference for the package, and use the resulting generated property `$(PkgSystem_Buffers)`.

```suggestion
    <EmbeddedResource Include="$(PkgSystem_Buffers)\lib\net461\System.Buffers.dll">
```

Reply:
@sharwell I just tried, this but it doesn't work.  It looks like there are some $pkg variables being created, but only for a subset of packages.

<img width="179" alt="image" src="https://user-images.githubusercontent.com/6210322/170587041-115d4b4f-4da6-48bf-b101-ce3b640008ed.png">

Do you know if there's something that I need to do other than add a `<PackageReference/>`?